apps/admin-gui/src/app/users/components/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,27 @@
+<div *ngIf="user !== undefined" class="container-fluid pl-xl-5 pr-xl-5 user-theme">
```

Comment:
I think that loading to this component is missing (you were doing the PR for adding loading to overview and headlines i believe)

Reply:
There are no api requests in this component so loading isn't needed.